pkg/substitution/substitution.go

```diff
@@ -44,6 +44,22 @@ func ValidateVariable(name, value, prefix, locationName, path string, vars sets.
 	return nil
 }
 
+func ValidateVariableP(value, prefix string, vars sets.String) *apis.FieldError {
```

Comment:
This is a temporary *simplified* function that would replace `ValidateVariable`. Didn't change `ValidateVariable` because it is used elswhere :upside_down_face: